.github/workflows/ci.yml

```
@@ -97,7 +97,7 @@ jobs:
         uses: actions/checkout@08c6903cd8c0fde910a37f88322edcfb5dd907a8 # v5
         with: { set-safe-directory: false }
       - name: Install zig
-        uses: korandoru/setup-zig@c1a533bcbce8c31ca826fd771dfce53deaf1ca77 # v1
+        uses: korandoru/setup-zig@3d137e800ac114393177819328e565c3adc75210 # v1
```

Action got archived and now links to this action

```suggestion
        uses: mlugg/setup-zig@8d6198c65fb0feaa111df26e6b467fea8345e46f # v2.0.5
```